include/qpdf/QUtil.hh

```diff
@@ -94,6 +94,11 @@ namespace QUtil
     QPDF_DLL
     FILE* safe_fopen(char const* filename, char const* mode);
 
+#ifdef _WIN32
```

Comment:
We can't have anything in the API be influenced by a preprocessor definition or have any public function that is not documented.